Octokit.Tests/Clients/StarredClientTests.cs

```diff
@@ -346,19 +345,31 @@ public class TheGetAllForUserMethod
         public class TheGetAllStargazersForRepoMethod
         {
             [Fact]
-            public void RequestsCorrectUrl()
+            public async Task RequestsCorrectUrl()
```

Comment:
he is saying that using `await` is the "correct" way, and we may just be lucky that we havent seen a timing issue when not using `await`